packages/web-components/src/horizontalbarchart/horizontalbarchart.ts

```diff
@@ -293,15 +296,28 @@ export class HorizontalBarChart extends FASTElement {
 
     let tooltip: any;
 
-    containerDiv
+    const chartTitleDiv = containerDiv.append('div').attr('class', 'chartTitleDiv');
+    chartTitleDiv
+      .append('div')
       .append('span')
       .attr('class', 'chartTitle')
       .text(data?.chartTitle ? data?.chartTitle : '');
 
+    const hideNumber = this.hideRatio === undefined ? false : this.hideRatio;
+
+    const showRatio = this.variant !== Variant.AbsoluteScale && !hideNumber && data!.chartData!.length === 2;
```

Comment:
enable this only if variant === part to whole

Reply:
Okay

Reply:
So by current condition also, it will get enabled when variant === part to whole

Reply:
it wont when you add 1 more variant

Reply:
Yes okay